Sources/TuistCore/Utils/BinaryLocator.swift

```diff
@@ -45,6 +45,16 @@ public final class BinaryLocator: BinaryLocating {
             bundlePath,
             bundlePath.parentDirectory,
             bundlePath.appending(try RelativePath(validating: "vendor")),
+            /**
+                == Homebrew directory structure ==
+                x.y.z/
+                   bin/
+                       tuist
+                   share/
+                       tuist/
+                           vendor
+                */
+            bundlePath.parentDirectory.appending(try! RelativePath(validating: "share/tuist")),
```

Comment:
```suggestion
            bundlePath.parentDirectory.appending(try RelativePath(validating: "share/tuist")),
```

This method already throws, so no reason for the `!`?